dts/bindings/xen/xen,xen-4.15.yaml

```diff
@@ -1,11 +1,5 @@
-description: Xen Platform Control Registers
+description: Xen Platform Control Registers v4.15
 
 compatible: "xen,xen-4.15"
```

Comment:
Probably we can drop exact version at all, just use "xen,xen"

Reply:
yes we can